demosplan/DemosPlanCoreBundle/Logic/Statement/AssessmentTableExporter/AssessmentTableZipExporter.php

```diff
@@ -110,18 +114,55 @@ private function getAttachmentsOfStatements(array $statementIds): array
     {
         $files = [];
         $index = 0;
+
+        $parameters = [
+            'procedureId' => $this->currentProcedureService->getProcedure()->getId(),
+            'anonymous'   => false,
+            'exportType'  => 'statementsOnly',
+            'template'    => 'portrait',
+            'original'    => true,
+            'viewMode'    => 'view_mode_default',
+        ];
+        // set file attachments if present:
         foreach ($statementIds as $statementId) {
             $statementAttachments = $this->statementService->getFileContainersForStatement($statementId);
-            $files[$index] = [];
+            $files[$index] = ['attachments' => [], 'originalAttachment' => null];
             foreach ($statementAttachments as $statementAttachment) {
-                $files[$index][] = $statementAttachment->getFile();
+                $files[$index]['attachments'][] = $statementAttachment->getFile();
+            }
+            // set the stn attachment:
+            // if present just take the given one.
+            $files[$index]['originalAttachment'] = $this->getOriginalAttachment($statementId);
+            if (null === $files[$index]['originalAttachment']) {
+                // if not present yet, invoke the pdfCreator and create an original-stn-pdf to use instead
+                $parameters['statementId'] =
+                    $this->statementService->getStatement($statementId)?->getOriginal()->getId();
+                $files[$index]['originalAttachment'] = $this->pdfExporter->__invoke(
```

Comment:
`__invoke` is not meant to be called directly

Reply:
done in [0e6e905](https://github.com/demos-europe/demosplan-core/pull/2444/commits/0e6e9057ec8a3988a461cea7b520254a1f7fe5ef)